packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -32,26 +32,25 @@ interface WaferMapArgs {
 
 const getDiesSet = (
     setName: string,
-    sets: WaferMapDie[][]
+    _sets: WaferMapDie[][]
 ): WaferMapDie[] | undefined => {
     const seed = 0.5;
     let returnedValue: WaferMapDie[];
     switch (setName) {
-        case 'set1':
-            returnedValue = sets[0]!;
+        case 'randomDies10':
+            returnedValue = generateWaferData(10);
             break;
-        case 'set2':
-            returnedValue = sets[1]!;
+        case 'randomDies100':
+            returnedValue = generateWaferData(100);
```

Comment:
should at least one data set be static for chromatic or it does not matter if it's random?

Reply:
Yeah, the default config in this story and every config in the matrix story needs to be static since Chromatic takes a snapshot every build.

Reply:
Sorry, I'm going to push back and request that all the storybook values be seeded random values like the goodDies and badDies examples that follow.

Storybook is useful for being able to share a link to the exact configuration you place the story to show someone a behavior. Using truly random values on each page load breaks that reproducibility.